README.md

```diff
@@ -28,13 +31,37 @@ export POLICY_PATH=/path/to/your/forseti-config-policies/repo
 # Generate a terraform plan.
 terraform plan --out=terraform.tfplan
 
+```
+
+### Terraform v0.11
+
+```
 # Validate the google resources the plan would create.
 terraform-validator validate --policy-path=${POLICY_PATH} ./terraform.tfplan
 
 # Apply the validated plan.
 terraform apply ./terraform.tfplan
 ```
 
+### Terraform v0.12
+
+For 0.12 Terraform release validator required plan exported in JSON format
+
+```
+# Plan JSON representation. 
+terraform show -json ./terraform.tfplan > ./terraform.tfplan.json
+
+# Validate the google resources the plan would create.
+terraform-validator validate --policy-path=${POLICY_PATH} ./terraform.tfplan.json
```

Comment:
I think we want to have the user specify json input / terraform version with explicit flags (eg --format=json --tf-version=0.12) since we will likely need to distinguish the version for future terraform releases.

Reply:
I wonder if we can auto-detect?

Reply:
@briantkennedy  I was also thinking about that

Reply:
> I wonder if we can auto-detect?
Currently it detected by file extension (.json or .tfplan) also it's not an issue to add programmatic check where input is "json" or "tfplan" format, but it will not work in case if user will pass corrupted file

Reply:
@briantkennedy @morgante  I think 2 params is overkill because TF12 support only json and TF11 only binary, are you ok if we'll proceed with only --tf-version=0.12?

Reply:
Yes, I think proceeding with a TF version flag makes sense.

Reply:
In that case let's go with --tf-version and check the file extension.

Reply:
@briantkennedy @morgante done, please review, p.s. tf-version is required paremeter